src/utils/round.ts

```diff
@@ -5,8 +5,13 @@
  * @returns The rounded value.
  */
 export function round(value: number): number {
-  if (value % 0.5 !== 0) {
-    return Math.round(value);
+  const integer = value >> 0;
```

Comment:
What about `Math.trunc(value)` ? To avoid unnecessary conversion to a 32-bit integer.

Reply:
You know I like bit shifts ... but I'm wrong !
Using `Math.trunc` is 2 times faster !!!!

With my bit shift: 10.719s, with Math.trunc: 5.952s. Clamped array:  5.302s